js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "reakt-native-toolkit",
-  "version": "0.16.0",
+  "version": "0.17.0.snapshot",
```

Comment:
keep release version

Reply:
Version 0.16.0? Wouldn't we update this to a release version after merge?

Reply:
We update after merge